### DISCUSSION TITLE
find files doesn't work when the file name contains some special characters

### DISCUSSION THREAD

**Discussion by varunbpatil** [ibhagwan/fzf-lua#1650] [Q&A] [answered]
Coming from this discussion in lazyvim - https://github.com/LazyVim/LazyVim/discussions/5262#discussioncomment-11680032

Using fzf-lua.

1. Create a file named foo::bar.
2. <leader>ff to open find files.
3. search for the above file.

The error I get is "Cannot stat file foo". This used to work earlier with telescope.nvim.

**Comment by ibhagwan** [comment, marked as answer]
That was a bug, should be fixed with https://github.com/ibhagwan/fzf-lua/commit/d99996b5838a30f4725e584bac5e8a0194e5a2a6
